BackgroundChanger.js

```diff
@@ -0,0 +1,31 @@
+const TIME = 10;
+
+function generateColors() {
+  const colors = {};
+  const randomColor = () => {
+    const r = Math.floor(Math.random() * 256);
+    const g = Math.floor(Math.random() * 256);
+    const b = Math.floor(Math.random() * 256);
+    return `rgb(${r},${g},${b})`;
+  };
+
+  for (let i = 0; i < TIME; i++) {
```

Comment:
In the challenge given it seemed that speed and total time were tied together. However, if the case is that they aren't tied directly together, then I would write it a different way.